daniddelrio/urc-seagrass
# Install React

Set up ReactJS and installed the ff packages:
1. [Material UI](https://material-ui.com/getting-started/installation/)
2. [React Leaflet](https://react-leaflet.js.org/docs/en/installation)
3. [Styled Components](https://styled-components.com/)

To test, do the ff steps:
1. Make sure you have Node installed
2. `cd` into the `client` folder, then run `npm install`
3. Run `npm start`, and see if something appears on your localhost:3000.
